maintenance/configFiles/rucio-server/alembic.ini

```diff
@@ -0,0 +1,68 @@
+# Copyright European Organization for Nuclear Research (CERN)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Vincent Garonne <vincent.garonne@cern.ch>, 2014
+# - Mario Lassnig <mario.lassnig@cern.ch>, 2019
+
+# A generic, single database configuration.
+
+# LDCS: this file should be placed in
+# /opt/rucio/etc
+
+[alembic]
+# path to migration scripts
+script_location = /usr/lib/python2.7/site-packages/rucio/db/sqla/migrate_repo/
```

Comment:
is there no py3 version? I'm worried for the fact that py2 is deprecated...

Reply:
Fair point, I am investigating if it is possible to use newer versions. Right now we are in the process of resuming an old setup, so maybe this will be improved. For now I believe the containers and systems that were used still had python2.7, and maybe this migration was already done, probably that line is not needed anymore. I'll report when I progress.

Reply:
The latest commit has all systems updated to python3.6.